zips/zip-0227.rst

```diff
@@ -322,19 +322,31 @@ For the ``IssueBundle``:
 
 **Note:** that the commitment is not included in the ``IssuanceAction`` itself. As explained below, it is computed later by the validators and added to the note commitment tree.
 
-Specification: Global Issuance State
-====================================
+Specification: Reference Notes and Global Issuance State
+========================================================
+
+Reference Notes
+---------------
+
+A reference note for a Custom Asset MUST be included by the issuer as the first Note in the Action of the Issuance Bundle where that Custom Asset is being issued for the first time.
+
+A reference note for a Custom Asset is an Issue Note where the value $\mathsf{v}$ is set to $0$, the Asset Base ($\mathsf{AssetBase}$) corresponds to that of the Custom Asset, and the recipient address $(\mathsf{d}, \mathsf{pk}_{\mathsf{d}})$ is set to the default diversified payment address derived from the all-zero Orchard spending key using the algorithm specified in §4.2.3 of the protocol specification [#protocol-orchardkeycomponents]_.
```

Comment:
what is "default diversified payment address" ?
We need " .. and the recipient address (d,pk_d) is derived from the all-zero Orchard spending key using the ... and **using 0 as the diversifier.**" Let's explicitly mention it even if "default diversified payment address" has the same meaning.

Alternatively,  "default diversified payment address (diversified with 0)". Here and again below

Also, let's add: Namely
```rust
[
    204, 54, 96, 25, 89, 33, 59, 107, 12, 219, 150, 167, 92, 23, 195, 166, 104, 169, 127, 13, 106,
    140, 92, 225, 100, 165, 24, 234, 155, 169, 165, 14, 167, 81, 145, 253, 134, 27, 15, 241, 14,
    98, 176,
]
```

Reply:
Added parentheses mentioning that it means the diversifier index is 0.

Added the values, like so: 
![image](https://github.com/user-attachments/assets/482d9c7d-2135-4ffc-892e-54d9aab706e2)